### PR TITLE
Delta Lake Diff: Enable feature

### PR DESCRIPTION
**Pull request by Jonathan-Rosenberg**
### Linked Issue

Closes #[5191](https://github.com/treeverse/lakeFS/issues/5191)

---

## Change Description

* Enable the Delta Lake diff functionality
* Remove `Experimental Delta Diff` button

## Bug fix

Before this PR, while hovering over the `show table changes` button on a compare row, the row was shrinking because the CSS behavior was removing the padding from the button. This PR fixes it.

### CODE REVIEW

File: webui/src/lib/components/repository/changes.jsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ export const TreeItemRow = ({ entry, repo, reference, leftDiffRefID, rightDiffRe
     const [afterUpdated, setAfterUpdated] = useState(""); // state of pagination of the item's children
     const [resultsState, setResultsState] = useState({results:[], pagination:{}}); // current retrieved children of the item
     const [diffExpanded, setDiffExpanded] = useState(false); // state of a leaf item expansion
-    const enableDeltaDiff = JSON.parse(localStorage.getItem(`enable_delta_diff`));
+    const disableDeltaDiff = JSON.parse(localStorage.getItem(`disable_delta_diff`));
```
Comment (nopcoder) [inline webui/src/lib/components/repository/changes.jsx:47]: 1. Use single or double quotes
2. JSON.parse looks like overkill
```suggestion
    const disableDeltaDiff = localStorage.getItem('disable_delta_diff') === '1';
```

**Review by eladlachmi — CHANGES_REQUESTED** [review]
Looks good—a couple of minor comments. Also, please take a look at the lint errors.

File: webui/src/styles/globals.css [hunk 2]
```diff
@@ -347,7 +347,7 @@ td.tree-path {
 }
 
 .table .change-entry-row .change-entry-row-actions .btn-link,
-.table .tree-entry-row .change-entry-row-actions .btn-link {
+.table .tree-entry-row .change-entry-row-actions .btn-link .btn.btn-link:hover {
```
Comment (nopcoder) [inline webui/src/styles/globals.css:350]: `.btn-link:hover` ?

File: webui/src/lib/components/repository/changes.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jonathan-Rosenberg) [reply webui/src/lib/components/repository/changes.jsx:47]: Thanks.
I'll remove that entirely.

**Review by eladlachmi — APPROVED** [review]
Thanks for implementing the fixes.
LGTM

File: webui/src/util/tablesUtil.jsx [hunk 3]
```diff
@@ -9,17 +9,11 @@ import {objects} from "../lib/api";
...
-    if (enableDeltaDiff) {
-        let response = await objects.list(repo.id, ref, entry.path + "_delta_log/")
-        if (response !== null && response.results.length !== 0) {
-            return true;
-        }
-    }
-    return false;
+    let response = await objects.list(repo.id, ref, entry.path + "_delta_log/")
+    return response !== null && response.results.length !== 0;
+
```
Comment (nopcoder) [inline webui/src/util/tablesUtil.jsx:18]: can remove the empty like before the end of the function

**Review by nopcoder — APPROVED** [review]
LGTM!

**Pull request merged**